templates/agenda/oe-agenda-session--oe-break.html.twig

```diff
@@ -8,6 +8,18 @@
 #}
 {% extends "@oe_whitelabel/agenda/oe-agenda-session--oe-default.html.twig" %}
 
+{% block hours %}
+  {{ pattern('icon', {
+    name: 'clock',
+    path: bcl_icon_path,
+    size: 'xs',
+    attributes: create_attribute({
+      class: ['mb-1', 'me-2'],
+    }),
+  }) }}
```

Comment:
To not repeat the icon here, I think we can use `macro`... We define it in the default template and  use `from `in order to use it in this template
https://twig.symfony.com/doc/3.x/tags/macro.html

Reply:
Added a macro in [3a8f0ac](https://github.com/openeuropa/oe_whitelabel/pull/268/commits/3a8f0aca720bdcd44eded9b2de01f82a3024fc2e) (we don't need from as we extend)